app/Helpers/render/layout.php

```diff
@@ -261,7 +261,7 @@ function RenderToolbar(): void
     echo "<form class='flex searchbox-top' action='/searchresults.php'>";
     // echo "Search:&nbsp;";
     echo "<input name='s' type='text' class='flex-1 searchboxinput' value='$searchQuery' placeholder='Search the site...'>";
-    echo "<input type='submit' value='🔎︎' title='Search the site'>";
+    echo "<button class='btn' title='Search the site'>🔎</button>";
```

Comment:
Adding a border to this button probably isn't desirable.

Prod:
![Screenshot 2023-08-17 at 5 40 53 PM](https://github.com/RetroAchievements/RAWeb/assets/3984985/2296f5b0-5163-4d09-985d-eda522ae0153)


PR:
![Screenshot 2023-08-17 at 5 40 10 PM](https://github.com/RetroAchievements/RAWeb/assets/3984985/332e9d0d-79a9-4e68-9fd3-91e5d073a261)

Reply:
Good catch! Fixed in https://github.com/RetroAchievements/RAWeb/pull/1774/commits/8bbcf4791a54bd8d54be2004ed3eb0ea4a955432